tests/testthat/test-DataWrangling.R

```diff
@@ -107,6 +107,70 @@ test_that("FindExpectedReferenceTreatment() returns first matching treatment whe
   expect_equal(!!result, "Placebo")
 })
 
+test_that(".FixColumnNameCases() fixes cases for continuous long data", {
+  data <- CleanData(read.csv("Cont_long.csv"))
+  names(data) <- c("sTuDy", "t", "n", "mEaN", "sD")
+  allowed_names = c("Study", "T", "N", "Mean", "SD")
+  
+  expect(all(!names(data) %in% allowed_names), failure_message = "Column names were not setup for the test correctly.")
+  
+  wrangled_data <- .FixColumnNameCases(data, "Continuous")
+  
+  expect_equal(colnames(wrangled_data), allowed_names,
+               label = format_vector_to_string(colnames(wrangled_data)),
+               expected.label = format_vector_to_string(allowed_names))
+})
+
+test_that(".FixColumnNameCases() fixes cases for continuous wide data", {
+  data <- CleanData(read.csv("Cont_wide.csv"))
```

Comment:
This test_that fails.

Reply:
Can you show me the error please? I have all tests passing locally, with no outgoing changes.

Reply:
If I run test_that.R everything passes, but if I run the test_thats individually, this one fails:

── Failure: .FixColumnNameCases() fixes cases for continuous wide data ─────────
c("Study", "T.1", "N.1", "Mean.1", "SD.1", "T.2", "N.2", "Mean.2", "SD.2", "T.3", "N.3", "Mean.3", "SD.3", "NA", "NA", "NA", "NA") not equal to c("Study", "T.1", "N.1", "Mean.1", "SD.1", "T.2", "N.2", "Mean.2", "SD.2", "T.3", "N.3", "Mean.3", "SD.3").
Lengths differ: 17 is not 13

Error: Test failed